### DISCUSSION TITLE
When will Node-Red 3.1 as hassio-addon be released?

### DISCUSSION THREAD

**Discussion by Wolk9** [hassio-addons/addon-node-red#1675] [General] [answered]
Hi Frenck and others,

I am very curious on when the new Node-Red release 3.1 is available in the hassio-addons. 

Keep up the good work!

**Comment by frenck** [comment, marked as answer]
> I am very curious on when the new Node-Red release 3.1 is available in the hassio-addons.

When it is ready.

../Frenck

**Reply by Wolk9** [reply]
How obvious. Haha. 

Well, then I won't bother you any longer and keep you from work.
